docs/docs/authoring.md

```diff
@@ -1,6 +1,6 @@
 # Authoring a markdown question
 
-This guide specifically applies to .md files which will be converted to the PrairieLearn (PL) format and uploaded to PL.
+This guide specifically applies to .md files which will be converted to the PrairieLearn (PL) format and uploaded to PL and will be converted to the public version and be uploaded to the public version of the Open Problem Bank.
```

Comment:
This may be better: 
This guide specifically applies to .md files which will be converted to the PrairieLearn (PL) format and uploaded to PL. The file will be converted to the public version and be uploaded to the public version of the Open Problem Bank.

Reply:
Alright, I will make that change